src/modules/site-v2/application.py

```diff
@@ -336,3 +347,70 @@ def handle_db_exceptions(err):
 #     db.session.remove()
 #   except Exception as db_err:
 #     logger.error(f'Exception closing sessions: {db_err}')
+
+
+def register_request_handlers(app):
+
+  # Redirect all URLs that point to this application to the host value in MODULE_SITE_HOST
+  # Individual API calls may fail if request body not copied over(?), but these should only be sent
+  # after initial page load, which will redirect to the correct host first
+  @app.before_request
+  def handle_redirects():
+    if request.host != MODULE_SITE_HOST:
+      return redirect(request.scheme + "://" + MODULE_SITE_HOST + request.full_path, code=307)

```

Comment:
side note: 307 is fine for testing, but we should look into 301 to make it permanent and cache-able by browsers.